deploy/003-deploy-VBep20.ts

```diff
@@ -59,8 +75,27 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     log: true,
     autoMine: true,
   });
+
+  await deploy("vBNB", {
+    contract: "VBep20Immutable",
```

Comment:
The contract of the vBNB market is not `VBep20Immutable`, but `VBNB`. Moreover, the underlying token is not WBNB

If we want to (partially) recreate the BNB mainnet deployment, we could:
1) rename VBNB.sol.archive to VBNB.sol
2) modify this deployment script to deploy a VBNB contract